components/layout/index.css

```diff
@@ -23,7 +23,7 @@
 }
 
 .layout-bar {
-  height: 3.5rem;
+  height: 4rem;
```

Comment:
I wander, why this did change?

Reply:
Because of the input element height.

Reply:
Input element height should be adjusted. You may use `size=small`

Reply:
Small didn't fit, i.e. it didn't look very well.